### PR TITLE
Fix/include outlines transformers integration

### PR DESCRIPTION
**Pull request by saattrupdan**
### Fixed
- Now includes the `transformers` integration with `outlines` directly in the code,
  which caused issues as they weren't part of the newest `outlines` release. When it
  does get included then we will import these as before.
- When evaluating OpenAI models we now do not perform any structured generation, as we
  do not have access to the logits.

### CODE REVIEW

**Comment by larsbun** [comment]
My installation from pip (12.2.0) breaks because of references to outlines.integrations. An indication of a package update would be great. Ty.

**Comment by saattrupdan** [comment]
> My installation from pip (12.2.0) breaks because of references to outlines.integrations. An indication of a package update would be great. Ty.

@larsbun  This should be fixed in version 12.2.1. Can you check if that fixes your issue?

**Comment by larsbun** [comment]
> > My installation from pip (12.2.0) breaks because of references to outlines.integrations. An indication of a package update would be great. Ty.
> 
> @larsbun This should be fixed in version 12.2.1. Can you check if that fixes your issue?

Works like a charm. TY.
